src/v/cluster/controller.cc

```diff
@@ -197,6 +200,8 @@ ss::future<> controller::start() {
             std::ref(_storage),
             std::ref(_as));
       })
+      .then(
+        [this] { return _drain_manager.invoke_on_all(&drain_manager::start); })
```

Comment:
I don't see an equivalent for `&drain::manager::stop`

Reply:
the service is stopped via `[this] { return _drain_manager.stop(); }` later on in the file. is that what you mean?

Reply:
Oh right, I forgot that knew how to stop the underlying service.

Reply:
> Oh right, I forgot that knew how to stop the underlying service.

I find this asymmetry with the api kind of annoying too